res/css/views/rooms/_AppsDrawer.pcss

```diff
@@ -313,7 +313,6 @@ limitations under the License.
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    font-size: $font-16px;
```

Comment:
This is because applying `justify-content: center` to `mx_AppPermission` makes it impossible to scroll to the top of the small pane.